lib/ec2imgutils/ec2utils.py

```diff
@@ -166,11 +199,23 @@ def get_from_config(account, config, region, entry, cmd_line_arg):
             raise EC2AccountException(msg % cmd_line_arg)
 
         account_name = generate_config_account_name(account)
-        try:
-            value = config.get(account_name, entry)
-        except Exception:
-            msg = 'Unable to get %s value from account section %s'
-            raise EC2AccountException(msg % (entry, account))
+        if entry in ['secret_access_key', 'access_key_id']:
```

Comment:
This is more of a style thing from my point of view. But why treat these two values special here? I.e. it could just as easily be:

```
try:
    value = config.get(account_name, entry)
except Exception:
    if entry in ['secret_access_key', 'access_key_id']:
        .....
```
This is more expressive and that it shows we are going down the "unexpected" code path, i.e. we expect the values to be in the default (.ec2utils.conf) configuration file and consider the aws-cli config file as a fall back path.

Again, this is more of a style issue, so I am not going to insist on this type of change.

Reply:
Thanks, I will switch this.